send-email/README.md

```diff
@@ -0,0 +1,60 @@
+---
+page_type: sample
+languages:
+- javascript
+- nodejs
+products:
+- azure
+- azure-communication-services
+---
+
+
+# Email Sample
+
+## Overview
+
+This is a sample email application to show how we can use the `@azure/communication-email` package to build an email experience.
+This sample sends an email to the required recipients of any domain using [Email Communication Services resource](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/email/create-email-communication-resource).
```

Comment:
"...required recipients of any domain using **an** [Email Communication Services resource]..."